src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/AmendCase.java

```diff
@@ -14,6 +17,65 @@ public void addTo(PageBuilder pageBuilder) {
             .complex(CaseData::getApplication)
                 .complex(Application::getMarriageDetails)
                     .optional(MarriageDetails::getDate)
+                .done()
+            .done()
+            .readonlyNoSummary(CaseData::getApplicationType,
+                "applicant1FirstName=\"NeverShow\"")
+            .complex(CaseData::getApplicant1)
+                .optional(Applicant::getFirstName,
+                    JOINT_APPLICATION_CONDITION,
+                    null)
```

Comment:
Is there an optional method where null argument is not needed?

Reply:
good point it would be nice to add it rather than passing null everywhere

Reply:
Replace null with NO_DEFAULT_VALUE